plugins/generator-1.20.1/forge-1.20.1/templates/block/block.java.ftl

```diff
@@ -204,7 +206,14 @@ public class ${name}Block extends
 	    	.setValue(AXIS, Direction.Axis.Y)
 	    	</#if>
 			<#list data.customProperties as prop>

```

Comment:
This got me wondering, why don't you just use initCustomBlockStateProperties here too, code seems identical?